src/components/HTMLEngineProvider/HTMLRenderers/NextStepsEmailRenderer.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Text from '@components/Text';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type NextStepsEmailRendererProps = {
+    tnode: {
+        data: string;
+    };
+};
+
+function NextStepsEmailRenderer({tnode}: NextStepsEmailRendererProps) {
+    const styles = useThemeStyles();
+
+    return <Text style={[styles.breakWord, styles.textLabelSupporting, styles.textStrong]}>{tnode.data}</Text>;
+}
+
+NextStepsEmailRenderer.displayName = 'NextStepsEmailRenderer';
+
+export default NextStepsEmailRenderer;
+export type {NextStepsEmailRendererProps};
```

Comment:
why this type is exported ? is it used somewhere else?

Reply:
Not actually, so I shouldn't export. Thank you!

Reply:
Done - d601867.